.bazelrc

```diff
@@ -1,2 +1,13 @@
-try-import %workspace%/user.bazelrc
```

Comment:
What's the reason for removing this?

Reply:
It's not the best practice to have user-specific Bazel rules inside a project as this can introduce various build flavors and potential build errors in the end. Each project should be hermetic meaning it should have project-specific rules and build the same no matter what the user environment is.

Reply:
I don't actually have user.bazelrc at the moment and can't remember why I wanted it, so I guess we can remove it for now, and if some justified reasons come up in the future we can bring it back.